projects/fsastorefrontlib/src/cms-components/checkout/components/payment-method/payment-method.component.html

```diff
@@ -10,18 +10,20 @@ <h4 class="mb-4">
         </h4>
         <form>
           <div class="form-check" *ngFor="let type of paymentTypes$ | async">
-            <input
-              id="paymentType-{{ type.code }}"
-              class="form-check-input"
-              type="radio"
-              name="paymentMethod"
-              [attr.aria-checked]="type.code === (paymentType$ | async)"
-              (click)="changeType(type.code)"
-              [value]="type.code"
-              [checked]="type.code === (paymentType$ | async)"
-            />
-            <label class="pl-3 pt-1">
-              {{ 'fscommon.payment.' + type.code | cxTranslate }}
+            <label class="pointer">
```

Comment:
This is a small fix for Payment details page, letting user click on label thereby selecting corresponding radio button.